### ISSUE TITLE
1.3 Copy of Patterns is not possible

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#7108]
### Bug Summary
It is not possible to copy/paste patterns in Pattern-editor

#### Steps to reproduce
Insert a pattern in one Beat-line and make a pattern
Right-click new pattern
Select Copy (img)
Use _Create new_ pattern
Point in new empty Beat-line
Right-click
Select paste
Nothing happens

#### Expected behavior
<1.3 the copied pattern is inserted in the empty Beat

#### Actual behavior
Copy does not work

#### Screenshot
![image](https://github.com/LMMS/lmms/assets/6662066/110db82f-4b89-45b8-8fa6-ff92152e806e)

Image show the context-selection after right-click 

#### Affected LMMS versions
Windows >1.3-alpha

This is a disaster!
It is vital that this can be done, because it is the only way to _split_ BB-lines.
Is this also ruined in Master?

**Comment by michaelgregorius** [comment]
I can also reproduce this in master with the following step (using a template with a pattern track):
1. Open the pattern editor.
2. Add a pattern, e.g. a four on the floor.
3. Right click the pattern and select "Copy".
4. Create a new pattern.
5. Create another new pattern.
6. Select "Pattern 1" from the combo box.
7. Paste into the copied pattern.
8. Adjust the pattern so that you can differentiate it from the first pattern.
9. Press play. Nothing is played.
10. Now select "Pattern 2".

The pattern is shown as empty but the copied and adjusted pattern from "Pattern 1" is being played.

It seems that the pattern is always copied to the very last pattern, i.e. if you add three patterns the sound will play for "Pattern 3".

**Comment by zonkmachine** [comment]
@michaelgregorius What you see is a glitch after the fix for issue https://github.com/LMMS/lmms/issues/6756 :
https://github.com/LMMS/lmms/pull/6803 / e87d44b42ca3a29bbc0ade8313deb6271cd5e166

**Comment by zonkmachine** [comment]
@IanCaio You may want to have a look at this one. @michaelgregorius's method above is the one to follow as the OP is just the bug fixed in #6803.

**Comment by zonkmachine** [comment]
I don't think it was e87d44b42ca3a29bbc0ade8313deb6271cd5e166 that introduced the issue but before this there was no copy/paste at all so this issue was hidden.
